### PR TITLE
Added BesLighthouse and updated Genesis file

### PR DESCRIPTION
**Pull request by anilsingla**
Added BesLighthouse installation script and updated Genesis file.

### CODE REVIEW

File: beslab_genesis.yaml [hunk 1]
```diff
@@ -1,54 +1,356 @@
...
+# BESLAB_MODE defines the type of BESLAB deployment
+# Values: host, bare , lite
+#    host: Deploy the lab on a hosted VM launched by
+#          oah-bes-vm and vagrant on windows based system.
+#    bare: Deploy the lab on local system or remote system
+#          using Ansible playbooks.
+#    lite: Deploy lab on local system using only shell 
+#          scripts without use use of Ansible or any VM.
+###########################################################
+BESLAB_MODE: lite
```
Comment (asa1997) [inline beslab_genesis.yaml:46]: We are passing it during init.

File: beslab_genesis.yaml [hunk 2]
```diff
@@ -1,54 +1,356 @@
...
+ 
+###########################################################
+###########################################################
+## Code Collabortion platfrom configurations
+## application only for private or public type of 
+## installation.
+###########################################################
+###########################################################
+
+## Configurations for public code collab
```
Comment (asa1997) [inline beslab_genesis.yaml:101]: Should be driven throught a single common variable instead of public and private

File: beslab_genesis.yaml [hunk 3]
```diff
@@ -1,54 +1,356 @@
...
+#BESLAB_BES_TOOL:
+#  - besman
+#  - bes-dev-kit
+#  - BesLighthouse
+
+## Applicable only if BeSLighthouse is to be installed.
+###########################################################
+
+# BESLAB_BESLIGHTHOUSE_RELEASE_VERSION: 0.16.2
+# BESLAB_BESLIGHTHOUSE_RELEASE_URL: https://github.com/Be-Secure/BeSlighthouse/archive/refs/tags/0.16.2.tar.gz
```
Comment (asa1997) [inline beslab_genesis.yaml:315]: Should be uncommented, otherwise the code wont work. Instead of beslighthouse, should be dashboard

**Review by asa1997 — CHANGES_REQUESTED** [review]
@anilsingla Please see the comments.

File: beslab_genesis.yaml [hunk 4]
```diff
@@ -1,54 +1,356 @@
...
+# BESLAB_TYPE is type of lab to be installed. 
+# Values: personal, public, private.
+#    personal: Installs all the tools in personal system
+#              No separate code collab platform is used.
+#    public: Does use public code collab platform e.g
+#            github, gitlab etc.
+#    private: Does install private code collab platfrom
+#             on the organization infra.
+########################################################### 
+BESLAB_TYPE: personal
```
[the remark this answers is not in the document]
Comment (anilsingla) [reply beslab_genesis.yaml:35]: To define the type of lab to be deployed. Will be used in future lab deployments.

File: beslab_genesis.yaml (hunk 1 above)
Comment (anilsingla) [reply beslab_genesis.yaml:46]: I am planning to provide it through Genesis file only.

File: beslab_genesis.yaml (hunk 2 above)
Comment (anilsingla) [reply beslab_genesis.yaml:101]: Public and private installations do have different requirements do need to separate.

**Pull request merged**